packages/sdk-themes/src/themes/monite.ts

```diff
@@ -945,6 +953,24 @@ export const defaultMoniteComponents: Components<Omit<Theme, 'components'>> = {
         'amount',
         'pay',
       ],
+      tab_filters: {
```

Comment:
mix of different Casings, let's stick to a single one. The theme object supports the camel case.
Or am I missing the reason for the snake case here?

Reply:
Certainly, I initially used the format mentioned in the ticket based on our customers’ requirements. However, I believe it’s important to prioritize following established conventions. We can also communicate with the customers to ensure alignment with this approach

Reply:
realized that tabFilters is a legacy from another task. In your case, it should be `cardFilters` or `summaryCardFilters`